std_msgs/Header header
APFinger/Proximity[] proximities
